src/sentry/api/helpers/group_index/update.py

```diff
@@ -255,7 +255,7 @@ def update_groups_with_search_fn(
     if group_ids:
         group_list = get_group_list(organization_id, projects, group_ids)
 
-    if not group_list:
+    if not group_list and not group_ids:
```

Comment:
I think this could be simplified to an `else` block if I'm reading this correctly, since `not group_list` is contingent on `group_ids`